src/Sulu/Bundle/ContentBundle/Resources/public/css/main.css

```diff
@@ -189,7 +189,7 @@
   line-height: 32px;
 }
 
-@media only screen and (max-width: 780px) {
+@media only screen and (max-width: 900px) {
```

Comment:
ah i know why ... the variables has changed in husky and that are copied to the admin und content bundle use this file ... perhaps in develop is this file not new builded!